collab/collab-error.js

```diff
@@ -3,7 +3,16 @@ export class CollabError extends Error {
     super(body);
     this.errorCode = errorCode;
     this.body = body;
-    Error.captureStackTrace(this, CollabError);
+    // Error.captureStackTrace is a v8-specific method so not avilable on
+    // Firefox or Safari
+    if (Error.captureStackTrace) {
+      Error.captureStackTrace(this, CollabError);
+    } else {
+      const stack = new Error().stack;
+      if (stack) {
+        this.stack = stack;
+      }
```

Comment:
This snippet is borrowed from google closure js library

https://github.com/google/closure-library/blob/6ae1f72f/closure/goog/debug/error.js#L37-L45